packages/go-bin/go-bin.pacscript

```diff
@@ -2,13 +2,14 @@ maintainer="Paul Cosma (saenai) <paul.cosma97@gmail.com>"
 
 name="go-bin"
 pkgname="go"
-version="1.17.8"
+version="1.18"
 description="Go is an open source programming language that makes it easy to build simple, reliable, and efficient software."
 url="https://go.dev/dl/go${version}.linux-amd64.tar.gz"
 breaks="${pkgname} ${pkgname}-git ${pkgname}-app ${pkgname}-deb golang golang-git golang-deb golang-bin golang-app golang-go"
 gives="golang-go"
 replace="golang-go"
-hash="980e65a863377e69fd9b67df9d8395fd8e93858e7a24c9f55803421e453f4f99"
+hash="e85278e98f57cdb150fe8409e6e5df5343ecb13cebf03a5d5ff12bd55a80264f"
+repology=("project: go" "repo: arch" "subrepo: community" "visiblename: go")
```

Comment:
```suggestion
repology=("project: go")
```

It works without additional filters, read [the tale of caution](https://github.com/pacstall/pacup/wiki/Wiki#a-tale-of-caution)